engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -843,11 +843,37 @@ object Main {
     */
   val defaultLogLevel: LogLevel = LogLevel.Error
 
+  var sampler : org.netbeans.modules.sampler.Sampler = org.netbeans.modules.sampler.Sampler.createSampler("LanguageServer")
```

Comment:
I feel like it is worth moving the sampler logic into a separate component to be able to do something like
```
val sampler = new Sampler(...)
sampler.start(duration = 10.seconds)
```